.github/workflows/wp-scripts-lint.yml

```
@@ -82,6 +94,18 @@ jobs:
               .parse(process.env.ENV_VARS)
               .forEach(envVar => core.exportVariable(envVar.name, envVar.value));
 
+      - name: Set up SSH
+        if: ${{ env.GITHUB_USER_SSH_KEY != '' }}
+        uses: webfactory/ssh-agent@v0.8.0
+        with:
+          ssh-private-key: ${{ secrets.GITHUB_USER_SSH_KEY }}
+
+      - name: Setup Git
```

```suggestion
      - name: Set up Git
```
Could you please fix this here as well, according to the pathfinder rules?
https://github.com/inpsyde/reusable-workflows/blob/f0fbb4ebdb4628bb40f9a949c3646720875e0e6a/.github/workflows/build-and-push-assets.yml#L116
Thanks!